docs/index.rst

```diff
@@ -22,6 +22,7 @@ HDL Reference Designs
 
    AD469X-FMC <projects/ad469x_fmc/index>
    AD719X-ASDZ <projects/ad719x_asdz/index>
+   AD5766-SDZ <projects/ad5766_sdz/index>
```

Comment:
Could you please move this one line before AD719X? Just to keep the alphabetical order.

Reply:
Done.